server/index.js

```diff
@@ -0,0 +1,71 @@
+var express = require('express');
+var app = express();
+var path = require('path');
+//var http = require('http').Server(app);
+
+
+app.use(require('./static.routing'));
+
+var gameConfig = require('./../config.json');
+
+var quadtree = require('simple-quadtree');
+var tree = quadtree(0, 0, gameConfig.width, gameConfig.height);
+
+var men = [];
+var sockets = {};
+
+function addMan (manToAdd) {
+	men.push(manToAdd);
+	tree.put(manToAdd);
+}
+
+function removeMan (manToRemove) {
+	var removeIndex = men.indexOf(manToRemove);
+	men.splice(removeIndex, 1);
+	tree.remove(manToRemove);
+}
+
+function moveLoop () {
+	for (var i = 0; i < men.length; i++) {
+		trackMan(men[i]);
+	}
+}
+
+function trackMan (man) {
+	moveMan(man);
+
+	tree.clear();
+	men.forEach(tree.put);
+}
+
+function moveMan (man) {
+
+}
+
+
+app.get('/*', function (req, res) {
+    res.sendFile(path.join(__dirname, '/../public/index.html'));
+});
+
+
+var server = app.listen(3030, function () {
+  console.log('Example app listening on port 3030!');
+});
+
+
+var io = require('socket.io').listen(server);
+io.on('connection', function (socket) {
+
+	console.log('a user connected!');
+
+	socket.on('disconnect', function () {
+		console.log('im disconnecting')
+	});
+
+	// to test if sockets work
+	setInterval(function(){
+		socket.emit('date', {'date': new Date()});
+	}, 1000);
+
+
```

Comment:
It's probably best not to push this stuff to master and suddenly force everyone else to make changes until we have a better grasp so we can make small changes bit by bit. 

While I agree that the static routing needs to be updated, that will have to be a small change by itself so it will be easier for everyone else to stay in sync.